### ISSUE TITLE
sendmail support?

### ISSUE THREAD

**Issue by prologic** [damicon/zfswatcher#11]
There doesn't seem to be (_unless I'm mistaken_) a way to configure `zfswatcehr` to use a sendmail compatible binary for email notifications. e.g: [esmtp](http://esmtp.sourceforge.net/)

**Comment by snabb** [comment]
You are correct. The notification/logging options are currently limited to SMTP, syslog and file logging.
